### PR TITLE
New package: python-google-api-python-client-1.4.2

### PR DESCRIPTION
**Pull request by pbui**
Also includes the following new packages as dependencies:
- python-oauth2client-1.5.2
- python-pyasn1-modules-0.0.8
- python-rsa-3.3
- python-uritemplate-0.6

### CODE REVIEW

**Comment by Vaelatern** [comment]
Different packages, as a rule, should be different commits.

**Comment by pbui** [comment]
@Vaelatern So should I close this and resubmit multiple PRs?  I wasn't sure since python-google-api-python-client depends on multiple new packages.

**Comment by Vaelatern** [comment]
```
10:08 <pnutzh4x0r> so should I close that PR and resubmit each package as a separate PR?
10:08 <Vaelatern> But each package should be a commit.
10:08 <Vaelatern> No, you can have 1 PR, just with multiple commits
10:08 <chris2> if you package deps too, put them in the same PR, different earlier commit
10:08 <pnutzh4x0r> ok, so a PR can have multiple commits
10:08 <Vaelatern> Yes 
10:09 <pnutzh4x0r> ok, thnx, i ll look into later
```

**Comment by pbui** [comment]
I have split each package into a separate commit as requested.

File: srcpkgs/python-google-api-python-client/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'python-google-api-python-client'
+pkgname=python-google-api-python-client
+version=1.4.2
+revision=1
+noarch=yes
+wrksrc="${pkgname#*-}-${version}"
+build_style=python-module
+python_versions="2.7 3.4"
+pycompile_module="google-api-python-client"
```
Comment (asergi) [inline srcpkgs/python-google-api-python-client/template]: `pycompile_module="apiclient googleapiclient"`

File: srcpkgs/python-google-api-python-client/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+license="Apache-2.0"
+homepage="http://github.com/google/google-api-python-client/"
+distfiles="${PYPI_SITE}/g/google-api-python-client/google-api-python-client-${version}.tar.gz"
+checksum=2cd7ccc797b0f5d1ef75ea1ebf749da47c4972daf534d22fd3c6b4dfbd9a88ee
+
+python3.4-google-api-python-client_package() {
+	noarch=yes
+	depends="python3.4 python3.4-httplib2 python3.4-oauth2client python3.4-uritemplate python3.4-six"
+	pycompile_version="3.4"
+	pycompile_module="google-api-python-client"
```
Comment (asergi) [inline srcpkgs/python-google-api-python-client/template]: Ditto.

File: srcpkgs/python-pyasn1-modules/template [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-pyasn1-modules'
+pkgname=python-pyasn1-modules
+version=0.0.8
+revision=1
+noarch=yes
+wrksrc="${pkgname#*-}-${version}"
+build_style=python-module
+python_versions="2.7 3.4"
+pycompile_module="pyasn1-modules"
```
Comment (asergi) [inline srcpkgs/python-pyasn1-modules/template]: `pycompile_module="pyasn1_modules"`

File: srcpkgs/python-pyasn1-modules/template [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+python3.4-pyasn1-modules_package() {
+	noarch=yes
+	depends="python3.4 python3.4-openssl python3.4-pyasn1"
+	pycompile_version="3.4"
+	pycompile_module="pyasn1-modules"
```
Comment (asergi) [inline srcpkgs/python-pyasn1-modules/template]: Ditto.

File: srcpkgs/python-rsa/template [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-rsa'
+pkgname=python-rsa
+version=3.2.3
```
Comment (asergi) [inline srcpkgs/python-rsa/template]: Please update to version 3.3. It's also a security update.

File: srcpkgs/python-rsa/template [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+noarch=yes
+wrksrc="${pkgname#*-}-${version}"
+build_style=python-module
+python_versions="2.7 3.4"
+pycompile_module="rsa"
+hostmakedepends="python-setuptools python3.4-setuptools"
+depends="python python-pyasn1"
+short_desc="Pure Python RSA implementation (Python2)"
+maintainer="Peter Bui <pnutzh4x0r@gmail.com>"
+license="BSD"
```
Comment (asergi) [inline srcpkgs/python-rsa/template]: `license="Apache-2.0"`

**Comment by pbui** [comment]
@asergi Thanks for the feedback, I've made the changes.

**Comment by asergi** [comment]
Merged manually with some minor tweaks. Thanks.

**Pull request closed without merging**